docs/cog_guides/audio.rst

```diff
@@ -0,0 +1,2555 @@
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]llsetup external 
+
+**Description**
+
+Toggle using external Lavalink servers.
+
+.. _audio-command-llsetup-host:
+
+""""""""""""
+llsetup host
+""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]llsetup host <host>
+
+**Description**
+
+Set the Lavalink server host.
+
+.. _audio-command-llsetup-info:
+
+""""""""""""
+llsetup info
+""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]llsetup info 
+
+.. tip:: Alias: ``llsetup settings``
+
+**Description**
+
+Display Lavalink connection settings.
+
+.. _audio-command-llsetup-java:
+
+""""""""""""
+llsetup java
+""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]llsetup java [java_path]
+
+**Description**
+
+Change your Java executable path
+
+Enter nothing to reset to default.
+
+.. _audio-command-llsetup-password:
+
+""""""""""""""""
+llsetup password
+""""""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]llsetup password <password>
+
+**Description**
+
+Set the Lavalink server password.
+
+.. _audio-command-llsetup-wsport:
+
+""""""""""""""
+llsetup wsport
+""""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]llsetup wsport <ws_port>
+
+**Description**
+
+Set the Lavalink websocket server port.
+
+.. _audio-command-local:
+
+^^^^^
+local
+^^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]local 
+
+**Description**
+
+Local playback commands.
+
+.. _audio-command-local-folder:
+
+""""""""""""
+local folder
+""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]local folder [folder]
+
+.. tip:: Alias: ``local start``
+
+**Description**
+
+Play all songs in a localtracks folder.
+
+.. _audio-command-local-play:
+
+""""""""""
+local play
+""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]local play 
+
+**Description**
+
+Play a local track.
+
+.. _audio-command-local-search:
+
+""""""""""""
+local search
+""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]local search <search_words>
+
+**Description**
+
+Search for songs across all localtracks folders.
+
+.. _audio-command-now:
+
+^^^
+now
+^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]now 
+
+**Description**
+
+Now playing.
+
+.. _audio-command-pause:
+
+^^^^^
+pause
+^^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]pause 
+
+**Description**
+
+Pause or resume a playing track.
+
+.. _audio-command-percent:
+
+^^^^^^^
+percent
+^^^^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]percent 
+
+**Description**
+
+Queue percentage.
+
+.. _audio-command-play:
+
+^^^^
+play
+^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]play <query>
+
+**Description**
+
+Play a URL or search for a track.
+
+.. _audio-command-playlist:
+
+^^^^^^^^
+playlist
+^^^^^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]playlist 
+
+**Description**
+
+Playlist configuration options.
+
+Scope info:
+​ ​ ​ ​ **Global**:
+​ ​ ​ ​ ​ ​ ​ ​ Visible to all users of this bot.
+​ ​ ​ ​ ​ ​ ​ ​ Only editable by bot owner.
+​ ​ ​ ​ **Guild**:
```

Comment:
There should be spaces inbetween the section on the `playlist` command so that it's not bunched up in one paragraph - should look like `playlist append` currently does.
<img width="738" alt="Screen Shot 2021-04-06 at 4 01 43 PM" src="https://user-images.githubusercontent.com/20862007/113788490-e482e380-96f1-11eb-8bf4-1daac12e68ae.png">

Reply:
Resolved in latest version